plugins/net_plugin/net_plugin.cpp

```diff
@@ -4377,23 +4377,23 @@ namespace eosio {
       for(auto listen_itr = listen_addresses.begin(), p2p_iter = p2p_addresses.begin();
           listen_itr != listen_addresses.end();
           ++listen_itr, ++p2p_iter) {
-         app().executor().post(priority::highest, [my=shared_from_this(), address = std::move(*listen_itr), p2p_addr = *p2p_iter](){
```

Comment:
Those two used highest priority; any particular reasons for that? Any side effect after moving them to the net thread pool as they will lose highest priority?

Reply:
They were highest priority so they would run before anything else on the main thread. Now they are run in the net thread pool and should also run right away.